starcatalogue/tasks.py

```diff
@@ -77,13 +78,40 @@ def download_fits(star_id):
         lightcurve.get_image_location()
 
 
+@shared_task
+def generate_star_json_files(star_id):
+    star = Star.objects.get(id=star_id)
+
+    if not star.fits:
+        return
+
+    ts = star.timeseries
+    if not ts:
+        return
+    ts_flux = Star.outlier_clip(ts["TAMFLUX2"])
+    ts_data = {
+        "data": {"x": ts.time.jd, "y": ts_flux},
```

Comment:
I'm not sure if these are actually directly JSON serialisable, but I guess let's try it and see!

Reply:
If not, can just convert to lists.